src/app.ts

```diff
@@ -13,14 +13,85 @@ interface TemperatureSummary {
   average: number
 }
 
+interface daySumary {
+  time: string
+  temperature: number
+  city: string
+  arrivalNumber: number
```

Comment:
This is a useless variable, you are only using it to set the order of the entry but in this case, you are not using it anymore (in another place)

Reply:
you're right eduardo, i'm going to delete that variable